CNV-30390: NUMA release notes for CNV 4.20 (#100837)
openshift/openshift-docs: virt/release_notes/virt-4-20-release-notes.adoc

// CNV-62399
* The descheduler profile `DevKubeVirtRelieveAndMigrate` has been renamed to `KubeVirtRelieveAndMigrate` and is now generally available. The updated profile improves VM eviction stability during live migrations by enabling background evictions and reducing oscillatory behavior. For more information, see xref:../../virt/managing_vms/advanced_vm_management/virt-enabling-descheduler-evictions.adoc#virt-configuring-descheduler-evictions_virt-enabling-descheduler-evictions[Configuring descheduler evictions for virtual machines].

// CNV-30390

> * vNUMA topology for VMs is now generally available (GA) and is tested and supported for production use. Before this update, this feature was provided as a Technology Preview, and KubeVirt VMs were not optimally configured for NUMA. This could lead to potential performance degradation due to ignorance of the host system's NUMA topology. With this update, the NUMA configuration for VMs is improved, ensuring better performance and optimal resource allocation. For more information, see xref:../../virt/managing_vms/advanced_vm_management/virt-numa-topology.adoc#virt-numa-topology[Working with NUMA topology for virtual machines].

Reviewer: ```suggestion
* vNUMA topology for VMs is now generally available (GA) and is tested and supported for production use allowing users to opt-in to a better NUMA configuration for VMs, with better performance and optimal resource allocation. For more information, see xref:../../virt/managing_vms/advanced_vm_management/virt-numa-topology.adoc#virt-numa-topology[Working with NUMA topology for virtual machines].
```
`vNUMA` configuration remains opt-in per VM so the fact this feature is now GA doesn't automatically enable it for all VMs and resolve all perf issues etc.

// CNV-62399
* The descheduler profile `DevKubeVirtRelieveAndMigrate` has been renamed to `KubeVirtRelieveAndMigrate` and is now generally available. The updated profile improves VM eviction stability during live migrations by enabling background evictions and reducing oscillatory behavior. For more information, see xref:../../virt/managing_vms/advanced_vm_management/virt-enabling-descheduler-evictions.adoc#virt-configuring-descheduler-evictions_virt-enabling-descheduler-evictions[Configuring descheduler evictions for virtual machines].

// CNV-30390

> * vNUMA topology for VMs is now generally available (GA) and is tested and supported for production use. By enabling this feature, you opt in to an improved NUMA configuration for VMs, with better performance and optimal resource allocation. For more information, see xref:../../virt/managing_vms/advanced_vm_management/virt-NUMA-topology.adoc#virt-NUMA-topology[Working with NUMA topology for virtual machines].

Reviewer 2: @abrennan89 do we always write "and is tested and supported for production use." when we write we are GA'ing a product.? i wasn't sure so thought of asking.

Reviewer 2: @abrennan89 probably asking your peers . I don't know.

+
For more information about selecting huge pages for an instance type, see xref:../../virt/creating_vm/virt-creating-vms-from-instance-types.adoc#virt-creating-vm-instancetype_virt-creating-vms-from-instance-types["Creating a VM from an instance type by using the web console"].

//CNV63575
* You can now easily identify if NUMA is enabled on your virtual machines. With this update, the `vNUMA` attribute is displayed in the VM details, showing NUMA status either next to the *CPU | Memory* section, or in the *InstanceType* section, depending on the VM creation method.

> 

Reviewer 2: @gouyang right . Do you mean dedicatedCPUPlacement ?Only cx1 provide that support.

> 

Reviewer 3: @geetikakay yes, `dedicatedCPUPlacement` can only be placed to VM created from template on web console.

I suggest remove `or in the *InstanceType* section, depending on the VM creation method` in the doc.

//CNV-61066
* Live migration fails if the VM name exceeds 47 characters. (link:https://issues.redhat.com/browse/CNV-61066[*CNV-61066*])

> // CNV-70330

Reviewer 2: in some places i see // CNV- and in some places it is //CNV-62357
